WordPress/Classes/Plugins/Views/PluginListItemView.swift

```diff
@@ -0,0 +1,112 @@
+
+    @ScaledMetric(relativeTo: .body) var descriptionFontSize: CGFloat = 14
+
+    private var iconURL: URL?
+    private var name: String
+    private var version: String
+    private var author: String
+    private var shortDescription: String
+    private var iconResolver: PluginIconResolver
+
+    init(plugin: InstalledPlugin, iconResolver: PluginIconResolver) {
+        self.iconURL = plugin.iconURL
+        self.name = plugin.name
+        self.version = plugin.version
+        self.author = plugin.author
+        self.shortDescription = plugin.shortDescription
+        self.iconResolver = iconResolver
+    }
+
+    var body: some View {
+        HStack(alignment: .top) {
+            CachedAsyncImage(urlResolver: iconResolver) { image in
+                image.resizable()
+            } placeholder: {
+                Image("site-menu-plugins")
+                    .resizable()
+            }
+            .frame(width: Self.iconSize, height: Self.iconSize)
+            .padding(.all, 4)
+
+            VStack(alignment: .leading, spacing: 0) {
+                Text(name)
+                    .lineLimit(1)
+                    .font(.headline)
+                    .foregroundStyle(.primary)
+
+                if !author.isEmpty {
+                    Text(Strings.author(author))
+                        .lineLimit(1)
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                }
+
+                Group {
+                    if shortDescription.isEmpty {
+                        Text(Strings.noDescriptionAvailable)
+                            .font(.system(size: descriptionFontSize).italic())
+                    } else if let html = renderedDescription() {
+                        Text(html)
+                    } else {
+                        Text(shortDescription)
+                            .font(.system(size: descriptionFontSize))
+                    }
+                }
+                .padding(.vertical, 4)
+
+                Text(Strings.version(version))
+                    .font(.caption)
+                    .foregroundStyle(.secondary)
+            }
+        }
+    }
+
+    func renderedDescription() -> AttributedString? {
+        guard var data = shortDescription.data(using: .utf8) else {
+            return nil
+        }
+
+        // We want to use the system font, instead of the default "Times New Roman" font in the rendered HTML.
+        // Using `.defaultAttributes: [.font: systemFont(...)]` in the `NSAttributedString` initialiser below doesn't
+        // work. Using a CSS style here as a workaround.
+        data.append(contentsOf: "<style> body { font-family: -apple-system; font-size: \(descriptionFontSize)px; } </style>".data(using: .utf8)!)
+
+        do {
+            let string = try NSAttributedString(
+                data: data,
+                options: [
+                    .documentType: NSAttributedString.DocumentType.html,
```

Comment:
I wouldn't risk using this on iOS on data that's coming from the web. It's slow, and it might hand or crash. For a short description like this, let's use plain text, and use a full-blown web view in the details screen.

Reply:
I don't think this description text would have long and complex html. It's mostly decorative, like bold or italic text. Using plain text would show things like `<strong>` in the content.

Reply:
> Using plain text would show things like <strong> in the content.

It will need to be sanitized with tags removed.